baas-tests/src/test/scala/com/ing/baker/baas/recipe/ReserveItemsInstance.scala

```diff
@@ -0,0 +1,34 @@
+package com.ing.baker.baas.recipe
+
+import cats.effect.{IO, Timer}
+import cats.implicits._
+import com.ing.baker.baas.recipe.CheckoutFlowEvents.ReserveItemsOutput
+import com.ing.baker.baas.recipe.CheckoutFlowIngredients.{Item, OrderId, ReservedItems}
+import com.ing.baker.baas.recipe.CheckoutFlowInteractions.ReserveItems
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+class ReserveItemsInstance(implicit timer: Timer[IO]) extends ReserveItems {
+
+  override def apply(orderId: OrderId, items: List[Item]): Future[ReserveItemsOutput] = {
+      IO.sleep(1.second)
```

Comment:
I would like to have these to emulate computation time, since these are big integration tests which should include asynchronousity to test for race conditions etc